src/bitcoin/mod.rs

```diff
@@ -172,9 +171,28 @@ impl BitcoinInterface for d::BitcoinD {
                     if block_height > 6 {
                         spent.push((*op, *txid, tx.block_time.expect("Spend is confirmed")))
                     }
-                } else {
-                    // TODO: handle the case where new transaction which txid is not the
-                    // coin spending_txid, spent the coin.
+                } else if !tx.conflicting_txs.is_empty() {
+                    for txid in tx.conflicting_txs.clone() {
```

Comment:
Do you need to `clone()`? Can't you just `as_ref()`?

Reply:
There is an immutable borrow on the `cache.get`, but you are right that it is a certainly better  way to do it